tests/scripts/abi.static.Test-Visible-Logarithmic.imagery_clean.sh

```diff
@@ -0,0 +1,20 @@
+# # # This source code is subject to the license referenced at
+# # # https://github.com/NRLMMD-GEOIPS.
+
+#!/bin/bash
+
+# Default values - if you do not have this exact test case available, call with available data files / sectors.
+
+# This exact test case required for valid comparisons - remove "compare_path" argument if running a different
+# set of arguments.
+geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_abi/data/goes16_20200918_1950/* \
+             --reader_name abi_netcdf \
+             --product_name Test-Visible-Logarithmic \
```

Comment:
This product is exclusively to test a new logarithmic plotting capability, the product isn't much use besides as an integration test.  But we wanted to ensure we were testing the new capability with standard data / readers that we had.